spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -65,20 +70,22 @@ public String send(final String topic, Message message) throws RuntimeException
 			try {
 				return Publisher.defaultBuilder(TopicName.create(projectId, topic))
```

Comment:
I just realised you suggested a CompositeMessageConverter [here](https://github.com/spring-cloud/spring-cloud-gcp/pull/6#issuecomment-303567611), I'll be sure to have a look at this soon!